stories/topics/con-saas-shape.adoc

```diff
@@ -10,15 +10,13 @@ Use the following to help guide your automation mesh implementation model:
 Hop nodes typically require minimum resources. 
 The shape of a hop node depends on the number of execution nodes connected to it. 
 Typically a virtual machine (VM) with 2 vCPUs and 2 GB RAM can route traffic for 2-4 execution nodes.   
-** Execution Nodes: {PlatformNameShort} minimum requirements are 4 vCPUs and 8 GB RAM for execution nodes. 
-** {PlatformNameShort} performance and scale testing showed that 2 m4.xlarge execution nodes could effectively automate against 200 managed active nodes constantly with enough remaining CPU and RAM as a buffer to prevent job failures in the event of a spike on those machines. 
-You can use this as a baseline to consider the right machine shapes for your environment when dealing with a CPU-heavy consistent automation load.
-You must monitor and tune the execution plane based on the needs of your organization’s automation.
+** For help with configuring your automation mesh see
+link:{BaseURL}/red_hat_ansible_automation_platform/{PlatformVers}/html/red_hat_ansible_automation_platform_automation_mesh_for_operator-based_installations/index[Red Hat Ansible Automation Platform Automation Mesh for operator-based installations].
```

Comment:
What version of AAP will SaaS use?
Maybe create a Jira to update the {PlatformVers} when it becomes necessary
I'm concerned that it might clash with Azure moving to 2.5. We can discuss this in the Clouds / docs sync

Reply:
Good question, I assumed 2.5 but I shouldn't be assuming anything.